FWCore/MessageService/plugins/MessageLogger.cc

```diff
@@ -15,8 +15,8 @@
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 
-#include "FWCore/MessageService/interface/MessageLogger.h"
-#include "FWCore/MessageService/src/MessageServicePSetValidation.h"
+#include "FWCore/MessageService/plugins/MessageLogger.h"
+#include "FWCore/MessageService/plugins/MessageServicePSetValidation.h"
```

Comment:
```suggestion
#include "MessageLogger.h"
#include "MessageServicePSetValidation.h"
```
to make it clear at a glance that these headers are internal to the same package, ratherthan being included from a different package's `plugins` directory.